redislite/__init__.py

```diff
@@ -27,12 +27,20 @@
     'package_metadata.json'
 )
 
-if os.path.exists(_metadata_file):  # pragma: no cover
+if os.path.exists(_metadata_file):
     with open(_metadata_file) as fh:
         _package_metadata = json.load(fh)
```

Comment:
Just wondering, is there any possibility that package_metadat.json might have invalid json format?